src/Bowling.js

```diff
@@ -0,0 +1,161 @@
+console.log("testtest");
+function BowlingGame() {
+  this.frames = 10;
+  this.PINS = 10;
+  this.ROLL = 2;
+  this.currentGo = 0;
+  this.currentScore = 0;
+  this.totalScore = null;
+  this.isStrike = false;
+  this.strikeCount = 0;
+  this.strikeScore = 0;
+  this.isSpare = false;
+  this.spareCount = 0;
+  this.spareScore = 0;
+  this.lastFrameExtra = false;
+  this.consecutiveStrikes = 0;
+  this.GAMEOVER_ERROR = "The game is over you cannot bowl again";
+
+  };
+
+  BowlingGame.prototype.bowl = function(pins) {
+    if ( this.frames === 0 ) throw Error(this.GAMEOVER_ERROR);
+
+    if (pins === 10) { this.isStrike = true;}
+    if (((this.currentScore + pins) === 10) && this.currentGo === 1) { this.isSpare = true;}
+
+    if ( this.frames === 1 ) { this.lastFrameCheck(pins);}
+
+    if (this.lastFrameExtra === true) {
+      this.lastFrame(pins);
+      } else {
+      this.currentMove(pins);
+      }
+  };
+
+
+  BowlingGame.prototype.currentMove = function(pins) {
```

Comment:
This function has too many statements. (14)<br>This function's cyclomatic complexity is too high. (6)